tutorials/snake/steps/step-3.md

```diff
@@ -13,23 +13,23 @@ We will assign some values to define what each cell of our grid can represent. A
 | 4     | tail    | `"S"`     |
 
 
-A snake as a head and a tail, and a body. We can represent it as a sequence of coordinates. The first element is the tail, the last one is the head, and everything in between is the body.
+A snake has a head and a tail, and a body. We can represent these entities as a sequence of coordinates. The first element is the tail, the last one is the head, and everything in between is the body.
 
 === "EDN"
 
     ```clojure linenums="1"
     [(int2 1 2) (int2 2 2) (int2 3 2) (int2 3 3) (int2 4 3)] >= .snake
     ```
 
-The fruit however occupies a single cell, so we just need one coordinate.
+The fruit, however, occupies a single cell - so we just need one set of coordinates.
 
 === "EDN"
 
     ```clojure linenums="1"
     (int2 4 4) >= .fruit
     ```
 
-We will render the grid using a [`(GUI.Table)`](https://docs.fragcolor.xyz/blocks/GUI/Table/) which simplifies the layout: we just need to push each value to the next column, and it will start a new row if needed.
+To simplify the layout we will render the grid using a <a href="https://docs.fragcolor.xyz/blocks/GUI/Table/" target="_blank">`GUI.Table`</a>. We just need to push each value to the next column and if there's no more space in that row a new row will be added automatically.
```

Comment:
No HTML

Reply:
Reverted to markdown link tag in latest commit.